.github/scripts/validate_binaries.sh

```diff
@@ -14,7 +14,7 @@ export PIP_DOWNLOAD_URL="https://download.pytorch.org/whl/cpu"
 export PIP_PREFIX=""
```

Comment:
Didn't you need to modify this one as well?

Reply:
Good catch!

Reply:
Happen to be a no-op, but yeah, I should have modified it.